projects/ddap-common-lib/src/lib/layout/menu/menu.component.ts

```diff
@@ -18,6 +18,9 @@ export class MenuComponent {
   isSandbox = false;
   @Input()
   logoUrl: string;
+  @Input()
+  logoLink?: string;
```

Comment:
I feel like those two properties are very similar in meanings and will lead to confusion. If you are going to anchor both elements under single `<a>` tag you can probably rename it to something more abstract or at least add comment. Perhaps: `headerLogoLink`